cmd/manager/cmd.go

```diff
@@ -260,6 +264,7 @@ func NewCmd(scheme *runtime.Scheme) *cobra.Command {
 	cmd.Flags().StringVar(&managerServiceName, "webhook-service-name", "kamaji-webhook-service", "The Kamaji webhook server Service name which is used to get validation webhooks, required for the TenantControlPlane migration jobs.")
 	cmd.Flags().StringVar(&managerServiceAccountName, "serviceaccount-name", os.Getenv("SERVICE_ACCOUNT"), "The Kubernetes Namespace on which the Operator is running in, required for the TenantControlPlane migration jobs.")
 	cmd.Flags().StringVar(&webhookCAPath, "webhook-ca-path", "/tmp/k8s-webhook-server/serving-certs/ca.crt", "Path to the Manager webhook server CA, required for the TenantControlPlane migration jobs.")
+	cmd.Flags().StringVar(&serviceAccount, "serviceAccount", "default", "The default serviceAccount that should be used by Kamaji to setup the required storage.")
```

Comment:
There's no need for an extra CLI argument: with an empty value, thanks to default values extracted from Kubernetes, we can keep the backward compatibility.

Reply:
Same as [this](https://github.com/clastix/kamaji/pull/441#discussion_r1572066759)